network/src/connection/network_connection.rs

```diff
@@ -91,8 +91,13 @@ impl NetworkConnection {
         (network_connection, process_connection)
     }
 
-    pub fn close(&self, ty: CloseType) -> bool {
-        self.peer_sink.close(ty, None).is_ok()
+    pub fn close(&self, ty: CloseType) {
+        self.peer_sink.close(ty, None);
```

Comment:
Returns a Result which is discarded. If Result is not required, this should be made explicit (e.g. `.ok()`).

Reply:
Errors now printed in log